physionet-django/templates/navbar_content.html

```diff
@@ -7,17 +11,15 @@
     </a>
   </li>
 
-  <li class="nav-item">
-    <a id="nav_publish" class="nav-link" href="{% url 'about_publish' %}">
-        Share
-    </a>
-  </li>
-
-  <li class="nav-item">
-    <a id="nav_about" class="nav-link" href="{% url 'about' %}">
-        About
-    </a>
-  </li>
+  {% for page in static_pg %}
+    {% if page.nav_bar %}
+      <li class="nav-item">
+        <a id="nav_{{ page.title | lower }}" class="nav-link" href="{{ page.url }}" >
```

Comment:
Aren't you missing the `underscore` filter here? All the other ids in this PR use it, now multi-word page titles will have spaces in their ID.
```suggestion
        <a id="nav_{{ page.title | underscore | lower }}" class="nav-link" href="{{ page.url }}" >
```

Since the `underscore` filter is only used in this combination then maybe it would be a good idea to just combine the functionality, like this:
```python
@register.filter
def snake_case(string):
    return string.replace(' ', '_').lower()
```
and use that in the templates instead of chaining`underscore | lower` each time

Reply:
Thanks @kshalot , I've added the `underscore` filter to the navbar template. We decided to keep it as its own filter so it could be used independent of `lower` if desired in the future.